app/Services/Model/Imp/SummitService.php

```diff
@@ -4160,4 +4176,63 @@ public function publishStreamUpdatesStartInXMinutes(int $minutes): void
             }
         }
     }
+
+    /**
+     * @param Summit $summit
+     * @param int $event_id
+     * @param array $payload
+     * @return SummitEvent
+     * @throws Exception
+     */
+    public function updateOverflowInfo(Summit $summit, int $event_id, array $payload): SummitEvent
+    {
+        Log::debug(sprintf("SummitService::updateOverflowInfo %s", json_encode($payload)));
+
+        return $this->tx_service->transaction(function () use ($summit, $event_id, $payload) {
+
+            $event = $this->event_repository->getByIdRefreshed($event_id);
+            if (is_null($event))
+                throw new ValidationException(sprintf("Event id %s does not exists,", $event_id));
+
+            $event->setOverflow(
+                $payload['overflow_streaming_url'],
+                boolval($payload['overflow_stream_is_secure']),
+                $this->generateOverflowStreamKey($event));
```

Comment:
this should be set on another call to be able to call setOverflow at factory level

Reply:
done